src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/help/Get-AzureRmDataLakeAnalyticsJobPipeline.md

```diff
@@ -110,7 +115,6 @@ System.Guid
 
 ## OUTPUTS
 
-### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.DataLakeAnalytics.Models.PSJobPipelineInformation, Microsoft.Azure.Commands.DataLakeAnalytics, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
@TaraMeyer sorry, I meant that we change the header to the content below it, so it would look something like

```
## OUTPUTS

### Microsoft.Azure.Commands.DataLakeAnalytics.Models.PSJobPipelineInformation

## NOTES
```

This will keep it consistent with the format of other cmdlets' output